.gitignore

```diff
@@ -11,4 +11,8 @@ project/plugins/project/
 
 # Vagrant
 .vagrant
-VERSION
\ No newline at end of file
+VERSION
+
+# IntelliJ IDEA
```

Comment:
IDE-specific and individual for developer items can be written in [global gitignore](http://stackoverflow.com/questions/7335420/global-git-ignore).